public/app/core/components/manage_dashboards/manage_dashboards.ts

```diff
@@ -67,6 +67,17 @@ export class ManageDashboardsCtrl {
 
   hasEditPermissionInFolders: boolean;
 
+  emptyListCta = {
+    title: "This folder doesn't have any dashboards yet",
+    buttonIcon: 'gicon gicon-dashboard-new',
```

Comment:
you do not need to define this here you can do it directly in the html file using single quotes inside the attribute quote like buttonIcon="'gicon gicon-dashboard-new'"

Reply:
Great point! Thanks @torkelo! 🚀